### ISSUE TITLE
add support for other devices

### ISSUE THREAD

**Issue by onlineapps-cloud** [Udj13/AGLoRa#2]
hi, very interesting project, it's possible to add support for devices like:
![image](https://github.com/Udj13/AGLoRa/assets/69077240/32dfcc92-2afa-4268-85ab-f4a846680043)
or `T-Beam S3-Core` or other compatible devices with GPS, esp and lora on board?
https://meshtastic.org/docs/hardware/devices/tbeam/
well tested devices with gps

**Comment by Udj13** [comment]
> hi, very interesting project, it's possible to add support for devices ....

Sounds interesting, I've already bought this module for experiments.

**Comment by onlineapps-cloud** [comment]
great to hear about this!

**Comment by boltm3** [comment]
how about meshtastic

**Comment by michaelbokov** [comment]
Добрый день, удалось ли собрать AGLoRa на подобном комбайнере? привлекает малый размер устройства, пусть и цена сборки выше ))?

**Comment by Udj13** [comment]
> Добрый день, удалось ли собрать AGLoRa на подобном комбайнере? привлекает малый размер устройства, пусть и цена сборки выше ))?

Да вариант то интересный, но если под esp32 переписывать то надо делать уже не на фреймворке Arduino а "по взрослому", с многозадачностью. Пока потихонечку приложение переделываю, чтобы можно было векторными картами offline пользоваться, за само устройство никак не возьмусь. Платки заказанные лежат, ждут, но это же хобби - не всегда получается так быстро как хотелось бы )

**Comment by Udj13** [comment]
> how about meshtastic

I really want to rewrite the code for them, but for now I'm more focused on the app.
In principle, it shouldn't be too difficult to adapt the code.

**Comment by marshalab** [comment]
Есть идея в замене arduino mcu + HC-05 на модуль esp32c3. Чтобы использовать esp32c3 как основной процессор и как Bluetooth Serial. Программа на смартфоне заточена под поток HC-05. Возможно ли эмулировать HC-05 средствами esp32c3 ? Это не заявка, а больше вопрос о сложности реализации к разработчику. 
В зависимости от версии bluetooth-модуля HC-05 и прошивки настроить можно разное количество параметров. Также AT-команд могут быть разные, у разных версий модулей и прошивок. Но основной набор команд неизменный для любой версии. 
UPD. https://github.com/Udj13/AGLoRa-full - нашел .... теперь изучаем заново )
Посмотрел - вы уже все реализовали! Единственно пока не понятно какое приложение использовать, старое или где то еще одно появилось. 
Вобщем отлично - пойду настраивать vscode.

**Comment by Udj13** [comment]
> Есть идея в замене arduino mcu + HC-05 на модуль esp32c3. Чтобы использовать esp32c3 как основной процессор и как Bluetooth Serial. Программа на смартфоне заточена под поток HC-05. Возможно ли эмулировать HC-05 средствами esp32c3 ? Это не заявка, а больше вопрос о сложности реализации к разработчику. В зависимости от версии bluetooth-модуля HC-05 и прошивки настроить можно разное количество параметров. Также AT-команд могут быть разные, у разных версий модулей и прошивок. Но основной набор команд неизменный для любой версии. UPD. https://github.com/Udj13/AGLoRa-full - нашел .... теперь изучаем заново ) Посмотрел - вы уже все реализовали! Единственно пока не понятно какое приложение использовать, старое или где то еще одно появилось. Вобщем отлично - пойду настраивать vscode.

Да, на c3 вариант работает, я сейчас на них себе устройства сделал.

По поводу приложение - на wiki есть ссылка на новое - lora radar (https://github.com/Udj13/AGLoRa/wiki/Lora-Radar-app), это прям альфа версия, но там фишка в том что это векторные карты которые кешируются на телефон. И плюс в нем я добавил вариант протокола с короткими переменными, чтобы меньше данных по BLE гонять

Если будете собирать себе устройство - чирканите мне на shlyagin@gmail.com свой акаунт в телеге, я вас в наш чатик добавлю по аглоре, там народу живой есть, могут что-то подсказать чтобы на грабли не наступать
И пожелания по новому приложению мне там высказывают )
